addons/info/src/panel/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import EmptyPanel from './empty_panel';
+
+/* select one from src/components/markdown/hljs-styles/
```

Comment:
We should either only include one, or allow a consumer to inject their own.

Having a whole bunch of themes in our codebase no-one can use unless they change OUR code, makes little sense, or am I reading this wrong?

Reply:
Is there any easy way to do it?
Actually, I should remove it!